giskard/ml_worker/utils/cache.py

```diff
@@ -0,0 +1,103 @@
+from typing import Any, Dict, List, Optional
+
+import hashlib
+import logging
+from pickle import PicklingError
+
+LOGGER = logging.getLogger(__name__)
+
+
+class SimpleCache:
```

Comment:
Why not using `functools.lru_cache` inside or even instead of this class?

Reply:
We don't want to be based on the whole arguments, but only on some of it. 
For example, we don't want to depends on client, ... only on params. So it will not easy. 
Also, for the cache to be usable with multiprocess, we need the start method, because the dict and list used are proxy of dict and list, created by SyncManager, not simple dict and list

Reply:
The main goal of this cache is to be able to shared elements between worker process